signal.go

```diff
@@ -22,6 +22,10 @@ var interruptHandlersDone = make(chan struct{})
 
 var simulateInterruptChannel = make(chan struct{}, 1)
 
+// signals defines the default signals to catch in order to do a proper
```

Comment:
I know it says the same thing in btcd now but could we remove "proper" (that word can mean anything, really) and just say that all of these signals are handled with clean shutdown, and that this also includes SIGINT on Unix with conditional compilation?